Entity/SettingInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Craue\ConfigBundle\Entity;
+
+/**
+ * @author Christian Raue <christian.raue@gmail.com>
+ * @copyright 2011-2017 Christian Raue

```

Comment:
I prefer to keep the year for now. It's not a big deal to perform a search and replace once a year. 😆